Podcast/UIImage+.swift

```diff
@@ -20,4 +20,12 @@ extension UIImage {
         UIGraphicsEndImageContext()
         return newImage!
     }
+
+    func withAlpha(_ alpha: CGFloat) -> UIImage? {
+        UIGraphicsBeginImageContextWithOptions(size, false, scale)
+        draw(at: .zero, blendMode: .normal, alpha: alpha)
+        let newImage = UIGraphicsGetImageFromCurrentImageContext()
+        UIGraphicsEndImageContext()
+        return newImage
```

Comment:
This Looks greatttt!!!!! FINALLY. Its so big that I can't really make too many comments because its hard to read through it all